cluster-up/cluster/kind/common.sh

```diff
@@ -14,12 +14,12 @@ function _wait_kind_up {
         sleep 10
     done
     echo "Waiting for dns to be ready ..."        
-    kubectl wait -n kube-system --timeout=12m --for=condition=Ready -l k8s-app=kube-dns pods
+    _kubectl wait -n kube-system --timeout=12m --for=condition=Ready -l k8s-app=kube-dns pods
```

Comment:
What's going on here?

Reply:
There is a function below called _kubectl that sets the correct kubectl. Actually, cluster/kind/common.sh isn't used by okd as far as I can tell. I'll remove this.